website/src/content/development/permissions.mdx

```diff
@@ -27,6 +27,41 @@ The `PERMISSIONS` variable contains a `View` and `Manage` properties, with the v
 
 You can then use the `PERMISSIONS` variable to reference the permission in the application code.
 
+<Info>
+
+This feature is available from version `21.21.0` onwards.
+
+When using additional permission groups, the unique group names should be provided to the `entryPointUriPathToPermissionKeys` function to generate the correct permission keys.
+
+The group names can be exported and referenced also in the Custom Application config file.
+
+```js title="constants.js" highlightLines="5-8, 12"
+import { entryPointUriPathToPermissionKeys } from '@commercetools-frontend/application-shell/ssr';
+
+export const entryPointUriPath = 'channels';
+
+export const groupNames = {
+  teamA: 'team-a',
+  teamB: 'team-b',
+};
+
+export const PERMISSIONS = entryPointUriPathToPermissionKeys(
+  entryPointUriPath,
+  [groupNames.teamA, groupNames.teamB]
+);
```

Comment:
See other suggestion about the group names

```suggestion
export const groupNames = {
  delivery: 'delivery',
  promotion: 'promotion',
};

export const PERMISSIONS = entryPointUriPathToPermissionKeys(
  entryPointUriPath,
  ['delivery', 'promotion']
);
```

Also, if we provide a TS example, we need to make sure to use the `as const` for the `groupNames`, so that we can pass the list using variable references

```ts
export const groupNames = {
  delivery: 'delivery',
  promotion: 'promotion',
} as const;

export const PERMISSIONS = entryPointUriPathToPermissionKeys(
  entryPointUriPath,
  [groupNames.delivery, groupNames.promotion]
);
```

Reply:
Thanks! That's a great catch!
:question: I'm still wondering how to approach the TS snippet. Is your point to integrate `@commercetools-docs/gatsby-theme-code-examples` as part of this PR?

Reply:
No that should probably be a follow up.